studio/config/docker/Dockerfile

```diff
@@ -39,10 +40,18 @@ RUN pip3 install --no-cache-dir --upgrade pip && \
     poetry config virtualenvs.create false
 RUN poetry install --no-root
 
+HEALTHCHECK --interval=30s --timeout=30s --start-period=5s --retries=3 \
+CMD curl -f http://0.0.0.0:8000/health || exit 1
+
 # copy optinist files
 COPY frontend/build /app/frontend/build
 COPY studio /app/studio
 COPY sample_data /app/sample_data
 COPY main.py alembic.ini ./
 
 EXPOSE 8000
+
+ENV HOST=0.0.0.0
```

Comment:
As I have told you in another message, there is no need to hard-code HOST and PORT here for ECS. (This is environment-dependent information and should be set in the ECS configuration (via management console))

I would like to see an explanation of how this setting was made necessary.